src/styles/_catalog.scss

```
@@ -14,6 +14,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: 0.3s;
```

you need to specify which properties you want to animate
```suggestion
    transition: transform 0.3s color 0.3s;
```